### DISCUSSION TITLE
每次固件升级后阿里云webdav都不能自动启动，手动更新token也不行了

### DISCUSSION THREAD

**Discussion by copperfieldd** [coolsnowwolf/lede#9684] [Bugs / 问题反馈]
2022-07-02T06:07:28.013141Z ERROR aliyundrive_webdav::drive: refresh token failed: HTTP status client error (405 Method Not Allowed) for url (https://www.aliyundrive.com/): Method POST not allowed
Error: get default drive id failed
2022-07-02T06:08:32.211195Z ERROR aliyundrive_webdav::drive: refresh token failed: HTTP status client error (405 Method Not Allowed) for url (https://www.aliyundrive.com/): Method POST not allowed
Error: get default drive id failed
2022-07-02T06:08:54.754968Z ERROR aliyundrive_webdav::drive: refresh token failed: HTTP status client error (405 Method Not Allowed) for url (https://www.aliyundrive.com/): Method POST not allowed
Error: get default drive id failed
![image](https://user-images.githubusercontent.com/25824246/176989001-3f98053b-5f0b-4a63-a49a-74acd2685abe.png)

**Comment by EnnawYang** [comment]
请更新1.5.1

**Reply by copperfieldd** [reply]
好的，谢谢

**Comment by daghy** [comment]
在系统--TTYD 终端--输入账号密码登录。
然后直接拷贝下面的命令先下载插件。
wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.5.1/aliyundrive-webdav_1.5.1-1_aarch64_generic.ipk wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.5.1/luci-app-aliyundrive-webdav_1.5.1_all.ipk wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.5.1/luci-i18n-aliyundrive-webdav-zh-cn_1.5.1-1_all.ipk
然后输入下面命令安装
opkg install aliyundrive-webdav_1.5.1-1_aarch64_generic.ipk opkg install luci-app-aliyundrive-webdav_1.5.1_all.ipk opkg install luci-i18n-aliyundrive-webdav-zh-cn_1.5.1-1_all.ipk

**Reply by copperfieldd** [reply]
好的，谢谢

**Reply by YuyangHou1230** [reply]
我更新到1.7.1了

**Comment by copperfieldd** [comment]
直接终端运行下面的命令，升级到最新版问题解决了。
wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.7.2/aliyundrive-webdav_1.7.2-1_aarch64_generic.ipk
wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.7.2/luci-app-aliyundrive-webdav_1.7.2_all.ipk
wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.7.2/luci-i18n-aliyundrive-webdav-zh-cn_1.7.2-1_all.ipk
opkg install aliyundrive-webdav_1.7.2-1_aarch64_generic.ipk
opkg install luci-app-aliyundrive-webdav_1.7.2_all.ipk
opkg install luci-i18n-aliyundrive-webdav-zh-cn_1.7.2-1_all.ipk
